ci/install_latest_cuda_toolkit.sh

```diff
@@ -2,17 +2,15 @@
 # Copyright (c) 2024, NVIDIA CORPORATION
 
 # Installs the latest CUDA Toolkit.
-# Supports CentOS 7 and Rocky Linux 8.
+# Supports Rocky Linux 8.
 
 yum update -y
 yum install -y epel-release
 
 OS_ID=$(. /etc/os-release; echo $ID)
-if [ "${OS_ID}" == "rocky" ]; then
-    yum install -y nvidia-driver
-else
+if [ "${OS_ID}" != "rocky" ]; then
     echo "Error: OS not detected as Rocky Linux. Exiting."
     exit 1
 fi
 
-yum install -y cuda-toolkit-12-5
+yum install -y nvidia-driver cuda-nvcc-12-5 cuda-cudart-devel-12-5 cuda-driver-devel-12-5 libnvjitlink-devel-12-5
```

Comment:
This is a much more minimal set of packages and should significantly speed up wheel builds in CI. I had meant to make this change long ago and never took the time for it until I started trying to run CI locally.

Reply:
Thank you for cleaning this up :slightly_smiling_face: